src/StreamFactory/GuzzleStreamFactory.php

```diff
@@ -3,6 +3,7 @@
 namespace Http\Message\StreamFactory;
 
 use Http\Message\StreamFactory;
+use GuzzleHttp\Psr7\Utils;
```

Comment:
the cs fixer orderes the imports alphabethically, please move this one to the top